test/e2e/multiple-resources/step0/inputs/rgd.yaml

```
@@ -0,0 +1,87 @@
+apiVersion: kro.run/v1alpha1
+kind: ResourceGraphDefinition
+metadata:
+  name: autoscaled-deployment-rgd
+spec:
+  schema:
+    apiVersion: v1alpha1
+    kind: AutoscaledDeployment
+    spec:
+      name: string | "some-app"
+      image: string | "nginx:latest"
+      minReplicas: integer | 1
+      maxReplicas: integer | 10
+      targetCPUUtilization: integer | 80
```

@barney-s i think you just caught a bug in kro :)
```suggestion
      minReplicas: integer | default=1
      maxReplicas: integer | default=10
      targetCPUUtilization: integer | default=80
```